### DISCUSSION TITLE
LGTMした記事から検索できないか

### DISCUSSION THREAD

**Discussion by Laplade** [increments/qiita-discussions#225] [要望・アイデア・不具合]
LGTMした記事からタグや入力文字で検索できる方法はありますか？
無いのなら追加してほしいです。
せっかくLGTMしたのに探しづらいです。

**Comment by kskwtnk** [comment]
@Laplade 
ご意見ありがとうございます！

すみません、現状はLGTMした記事を検索はできません 🙇 

後から参照しそうな記事の場合、ストック機能をご利用いただけますと幸いです
カテゴリー分けやキーワードで検索ができますので、LGTMよりも便利に探せるかと思います

ご意見いただいた内容とは違う返信になってしまい恐れ入りますが、ご理解いただけますと幸いです 🙇
